scripts/utils/cli-step.ts

```diff
@@ -0,0 +1,45 @@
+import { getCommand, OptionSpecifier, OptionValues } from './options';
+import { exec } from '../../code/lib/cli/src/repro-generators/scripts';
+
+const logger = console;
+
+const cliExecutable = require.resolve('../../code/lib/cli/bin/index.js');
```

Comment:
About what I mentioned before, this is where the validation should happen, else it will error:

![image](https://user-images.githubusercontent.com/1671563/181455783-10e941f2-7288-48cd-8e93-ab52e43358e8.png)


```suggestion
import dedent from 'ts-dedent';

let cliExecutable: string;
try {
  cliExecutable = require.resolve('../../code/lib/cli/bin/index.js');
} catch (e) {
  logger.error(dedent`
    🚨 Failed to find CLI executable. 
    Please make sure to build the Storybook packages (e.g. yarn bootstrap --prep) prior to running "yarn example."
  `);
  process.exit(1);
}
```

Reply:
or `throw e` rather than `process.exit(1)` if you'd like the message to also include the stack trace

Reply:
Thanks @yannbf. I think we will change this code to call directly into `lib/cli/src` so this point will become moot soon!